includes/Runtime.php

```diff
@@ -45,9 +45,16 @@ public function prepareRuntime() {
 				'title' => htmlspecialchars_decode( \get_bloginfo( 'name' ) ),
 			),
 			'admin_url'    => \admin_url(),
+			'adminUrl'		=> \admin_url(),
 			'base_url'     => \get_home_url() . '/index.php',
+			'homeUrl'     => \get_home_url() . '/index.php',
 			'capabilities' => $this->container->get( 'capabilities' )->all(),
 			'sdk'          => $sdk,
```

Comment:
In task description it says, "we should remove the sdk property and just let its contents live at the top level. "